content/A good way to manage your Python projects using Pyenv and Pipenv.md

```diff
@@ -0,0 +1,290 @@
+
+There you can see that after the `pipenv shell` command has been executed, the name of the virtual environment appeared between brackets at the beginning of the line. To deactivate the virtual environment, just run:
+
+```
+exit
+```
+
+
+### Managing Packages
+Ok, so you now know how to create, delete, activate and deactivate a virtual environment. Now let's see how to manage the packages inside them. First, don't forget to activate the virtual environment. To install a package, for example [django-drip-campaings](https://pypi.org/project/django-drip-campaigns/), you can execute:
+
+```
+pipenv install django-drip-campaigns
+```
+
+This command will:
+- Install the latest available version of that package.
+- Automatically update the Pipfile file adding the package.
+- Automatically update the Pipfile.lock file adding the package version and some other important metadata.
+
+Now the Pipfile file looks like this:
+
+```
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+django-drip-campaigns = "*"
+
+[dev-packages]
+
+[requires]
+python_version = "3.8"
+```
+
+As you can see, the package was installed under the `[packages]` section. As mentioned, in the `[dev-packages]` section you have to install the ones that you need only for development. For example, you want to have [flake8](https://pypi.org/project/flake8/) in your project, that is only required by the development process. To install it under the `[dev-packages]` section, execute:
+
+```
+pipenv install flake8 --dev
+```
+
+After that, you will see something like this in the Pipfile file:
+
+```
+[packages]
+django-drip-campaigns = "*"
+
+[dev-packages]
+flake8 = "*"
+```
+
+Also, what if you need a specific version of a package? For example, you need [Django](https://www.djangoproject.com/), not the latest but the 2.2 version. Then run:
+
+```
+pipenv install django==2.2
+```
+
+Now the Pipfile is updated and will resemble something like:
+
+```
+[packages]
+django-drip-campaigns = "*"
+django = "==2.2"
+
+[dev-packages]
+flake8 = "*"
+```
+
+In the `Django` package line, you don't see the "*", you instead see the specific version indicated in the command.
+If you need to uninstall a package, for example `django-drip-campaigns`, just run:
+
+```
+pipenv uninstall django-drip-campaigns
+```
+
+That command will uninstall the package, and remove the information from the Pipfile and Pipfile.lock files.
+Now suppose you need the latest version of `django` in your project(not the 2.2 version). To update a package to the latest version you have to:
+
+1. Change the `django = "==2.2"` line in the Pipfile by `django = "*"`.
+2. Then run:
+
+```
+pipenv update django
+```
+
+An alternative to these steps is to uninstall `django` and then install it again without specifying the version. Of course, you can repeat those steps in an analogous way, to downgrade a package version instead of upgrading it. 
```

Comment:
```suggestion
Alternatively, you can uninstall `Django` and then install it again without specifying the version. Of course, you can repeat those steps in an analogous way, to downgrade a package version instead of upgrading it. 
```